### DISCUSSION TITLE
Mysterious "ghost" contributor

### DISCUSSION THREAD

**Discussion by xparq** [community/community#45623] [Repositories] [answered]
### Select Topic Area

Bug

### Body

Similar perhaps to #22272 (unknown "impostor" committer), a phantom user appeared as a contributor to one of my repos (xparq/Args, created a few days ago):

![y-onodera01-WTF](https://user-images.githubusercontent.com/698307/215268640-dc966d63-1973-4a08-8c16-ce97d2b9a684.png)

_(Yeah, well, the real ghost is me on the picture, BTW... :) )_

Except, here the "contributor" did nothing: the [commits listed when clicking the name](https://github.com/xparq/Args/commits/main/Args.hpp?author=y-onodera01) are not his, but mine. 
Also, that profile is completely blank (ever since it was created many years ago), no activities there either.

Probably relevant: noticed late in the game that my current local Git setup was not fully configured, and had an "example.com" address for the email, so a few commits surely contained the wrong one.
But even then: GH could still correctly identify me as a contributor all along, despite the wrong email, and everything seemed perfectly normal, until something made it think that there is this "other guy".

Yet another possibly related clue, FTR: weeks ago I got a mention-notif. for a Chinese repo, where -- as far as I could check :) (even got it translated for context) -- I wasn't mentioned at all, or couldn't see any remotely recognizable reference. Since it looked like some general programming course material, could that be another stray "example.com" linkage?

(That domain in general should just be ignored in all these contexts, I guess.)

Thanks for everything, all the best!

--------
(UPDATE: Oh, now that I mentioned that mention..., just received a new mail about a message [there](https://github.com/krahets/hello-algo/discussions/149#discussioncomment-4802724), just for ref.)

**Comment by airtower-luna** [comment]
If you mean this commit https://github.com/xparq/Args/commit/204353a2e70cdadd856a1b8629528ed69afe543e, I don't see a second committer, but also the commit isn't recognized as yours (wrong email is the reason with high certainty). If that's not where you're looking, could you link the actual page?

**Comment by xparq** [comment]
I've linked the misidentified commits in the orig. descr.

So, the gist of the issue remains that such mistaken matches definitely shouldn't happen, when the email is _not simply shared_ by multiple profiles, but is of the standard _invalid_ domain of **`example.com`**.

Also: is there a way to clean these misidentified "contributors" from the repo?

Note: I guess profiles also shouldn't really be able to have invalid `...@example.com` addresses to begin with. _(BTW, by typing that address in the prev. sentence, autocomplete offered 4 other such users! :-o (but not my ghost, curiously).)_

**Comment by airtower-luna** [comment, marked as answer]
Peculiar, with that search by author you added I get a list of 5 commits, but *for none of the commits* the person is actually shown as author when I check the commit page (e.g. https://github.com/xparq/Args/commit/204353a2e70cdadd856a1b8629528ed69afe543e):

![grafik](https://user-images.githubusercontent.com/5814919/215882634-9e4dabe6-bc0f-4309-9647-b006c3bf2910.png)

Might be time for a bug report… :sweat_smile: https://support.github.com/contact/bug-report

**Comment by xparq** [comment]
@airtower-luna _"none of the commits the person is actually shown as author"_

Well, that's what I've been telling. :)

But thanks for the actual bug report form! (Actually I assumed this discussion forum itself is already a GH feedback funnel (because it kinda replaced the old official ones, as per some SO/SX googling...), but alas, it appears to not be.)
